third_party/2.7/itsdangerous.pyi

```diff
@@ -0,0 +1,153 @@
+# Stubs for itsdangerous (Python 2.7)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from datetime import datetime
+from itertools import izip
+from typing import Any, Callable, IO, MutableMapping, Optional, Text, Tuple, Union
+
+PY2 = ...  # type: bool
+text_type = unicode
+int_to_byte = chr
+number_types = (int, long, float)
+
+bytes_like = Union[bytearray, str]
```

Comment:
Hm... I wonder if even for 2.7 it's better to use bytes instead of str? Honestly I have no idea what this package does (and no time to research it) so if you don't think that's a good idea I'll take your word for it. If you do, maybe it should go into 2and3?